proxysql_tools/entities/galera.py

```diff
@@ -84,7 +85,8 @@ def refresh_state(self):
     def get_connection(self):
```

Comment:
The exception is handled through galera manager which is where this entity is used. I will add the expected exceptions to the docstring.